src/NServiceBus.NHibernate.Tests/ApprovalFiles/DDL.GatewayDeduplication.approved.txt

```diff
@@ -4,6 +4,6 @@
     
     create table GatewayDeduplication (
         Id NVARCHAR(255) not null,
-       TimeReceived DATETIME null,
+       TimeReceived DATETIME2 null,
```

Comment:
None of these DATETIME -> DATETIME2 changes could be considered a breaking change, could it?

Reply:
@DavidBoike I have no idea. That's a change in v5.0 of NH and it can be configured to use the previous one, but this is global configuration setting so I wouldn't do it. Do we need a test for this then?

For sake of reference, a SO question mentioning it https://stackoverflow.com/questions/32679529/nhibernate-and-sql-server-datetime2-and-supporting-both-sql-server-and-oracle

Reply:
Seems like datetime2 is larger range and higher precision than datetime and should accept datetime values just fine. Think we're OK.

Reply:
Is there any problem here around the fact that people will already have tables defined with `DATETIME` and now we'll be expecting `DATETIME2` instead?